docs/MO_DG/prepare_model/Model_Optimizer_FAQ.md

```diff
@@ -440,11 +440,11 @@ Keep in mind that there is no space between and inside the brackets for input sh
 
 #### Q75. What does the message "... elements of ... were clipped to infinity while converting a blob for node [...] to ..." mean? <a name="question-75"></a>
 
-**A** : This message may appear when the `--data_type=FP16` command-line option is used. This option implies conversion of all the blobs in the node to FP16. If a value in a blob is out of the range of valid FP16 values, the value is converted to positive or negative infinity. It may lead to incorrect results of inference or may not be a problem, depending on the model. The number of such elements and the total number of elements in the blob is printed out together with the name of the node, where this blob is used.
+**A** : This message may appear when the `--compress_to_fp16` command-line option is used. This option implies conversion of all the blobs in the node to FP16. If a value in a blob is out of the range of valid FP16 values, the value is converted to positive or negative infinity. It may lead to incorrect results of inference or may not be a problem, depending on the model. The number of such elements and the total number of elements in the blob is printed out together with the name of the node, where this blob is used.
```

Comment:
We left `--data_type` as a deprecated parameter, so the user can really apply it instead of `--compress_to_fp16`. And imagine that such a user will read this message about `--compress_to_fp16` which he/she didn't apply and now faces for the first time. It will confuse the user, the user will ignore this message or takes it as suspicious, not connected to the problem which the user experienced. It is better to address the initial error message to make it clear that the reason for that truncation is actually the parameter (`--data_type` or `--compress_to_fp16`).

In this FAQ please use `--compress_to_fp16 (or deprecated --data_type)`

Reply:
Done.